.tekton/run-acceptance-test.yaml

```diff
@@ -0,0 +1,146 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: run-acceptance-test
+  annotations:
+    pipelinesascode.tekton.dev/on-event: "[pull_request]"
+    pipelinesascode.tekton.dev/pipeline: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/pipelines/acceptance-tests.yaml"
+    pipelinesascode.tekton.dev/on-target-branch: "[master]"
+    pipelinesascode.tekton.dev/task-01: "git-clone"
+    pipelinesascode.tekton.dev/task: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/cleanup-cloud-resources.yaml"				
+    pipelinesascode.tekton.dev/task-1: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/send-gitlab-mr-test-results-to-dashboard.yaml"
```

Comment:
you have `task`, `task-01` and `task-1`... can we start with numbers and omit `task` without number?